<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>type_integrate_score</class>
 <widget class="QWidget" name="type_integrate_score">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>学年</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="semesteryearcombobox">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="majorcombobox_4">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>专业</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="majorcombobox">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="selectclass_2">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>班级</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="classcombobox">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>按学号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>按姓名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>查找</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/image/search (2).png</normaloff>:/image/search (2).png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>198</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>重新计算综合测评成绩</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/image/pencil.png</normaloff>:/image/pencil.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="integrate_score_widget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="locale">
          <locale language="Chinese" country="HongKong"/>
         </property>
         <property name="text">
          <string>刷新表格</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/image/refresh (2).png</normaloff>:/image/refresh (2).png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="modifyButton">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>修改</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/image/modify.png</normaloff>:/image/modify.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>撤销</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/image/refresh.png</normaloff>:/image/refresh.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>保存(&amp;S)</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/image/tick.png</normaloff>:/image/tick.png</iconset>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>该学年加减分项目</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="newAddButton">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>添加新项</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/image/add.png</normaloff>:/image/add.png</iconset>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteAddButton">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>删除该项</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/image/minus (2).png</normaloff>:/image/minus (2).png</iconset>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="modifyAddButton">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>修改</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/image/modify.png</normaloff>:/image/modify.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveAddButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>保存(&amp;S)</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/image/tick.png</normaloff>:/image/tick.png</iconset>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_8">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>查找</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/image/search (2).png</normaloff>:/image/search (2).png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="addMinusItemView">
        <property name="cursor" stdset="0">
         <cursorShape>UpArrowCursor</cursorShape>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addStudentItemButton">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="text">
        <string>添加至学生</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/image/down_alt.png</normaloff>:/image/down_alt.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteStudentItemButton">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>从学生删除</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/image/up_alt.png</normaloff>:/image/up_alt.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="infoBox">
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>基本信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="name">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>姓名</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="stunameEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>151</width>
          <height>21</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="stunum">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>学号</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="stunumEdit">
        <property name="minimumSize">
         <size>
          <width>151</width>
          <height>21</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QTableView" name="individual_additem_view">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
